rsconnect/documents/HW-0.Rmd/rpubs.com/rpubs/Publish Document.dcf

```diff
@@ -0,0 +1,7 @@
+name: Publish Document
+account: rpubs
+server: rpubs.com
+appId: https://api.rpubs.com/api/v1/document/155294/ff7dac3d47554c25979ad65ddf75e0dc
+bundleId: https://api.rpubs.com/api/v1/document/155294/ff7dac3d47554c25979ad65ddf75e0dc
+url: http://rpubs.com/cmernestein/155294
+when: 1456283481.06098
```

Comment:
Next time do not submit (commit and then push) files like this one. I only need the `.R` or `.Rmd` files